rampwf/tests/kits/boston_housing/problem.py

```diff
@@ -10,7 +10,8 @@
 # A type (class) which will be used to create wrapper objects for y_pred
 Predictions = rw.prediction_types.make_regression()
 # An object implementing the workflow
-workflow = rw.workflows.Regressor()
+# workflow = rw.workflows.Regressor()
+workflow = rw.workflows.SKLearnPipeline(file_name='regressor', is_proba=False)
```

Comment:
I would call it filename or fname. fname is the naming convention for readers like loadtxt